touchforms/backend/xformserver.py

```diff
@@ -236,6 +236,13 @@ def handle_request(content, server):
         elif action == xformplayer.Actions.GET_INSTANCE:
             xfsess = xformplayer.global_state.get_session(content['session-id'])
             return {"output": xfsess.output()}
+
+        elif action == 'get-instance-xml':
+            if 'session-id' not in content:
```

Comment:
adding `'get-instance-xml'` to the `xformplayer.Actions` would be nice. also, you do not need this check, it's done here: https://github.com/dimagi/touchforms/blob/tf-instance-viewer-feature-flag/touchforms/backend/xformserver.py#L159